compiler/parser/parser.peg

```diff
@@ -517,6 +518,15 @@ TailOp
       }, nil
     }
 
+OffsetOp
+  = OFFSET _ !EndOfOp count:Expr {
```

Comment:
No need for !EndOfOp here since an expression always follows OFFSET (unlike HeadOp and TailOp, where the expression is optional).

```suggestion
  = OFFSET _ count:Expr {
```

Reply:
Thanks for the clarification!